### PR TITLE
Fixes #213. Updating nav bar links and adding footer.

### LINKED ISSUE #213: Revisit navigation bar links and using "My"

### PR DESCRIPTION
**Pull request by lwrubel**

### CODE REVIEW

**Comment by lwrubel** [comment]
Footer and styling generally could use further attention, perhaps as part of #17

File: sfm/ui/templates/base.html [hunk 1]
```diff
@@ -55,38 +55,37 @@
...
-          <ul class="nav navbar-nav navbar-right">
-            {% if request.user.is_authenticated %}
-			  <li><a href="/ui/collections">My Collections</a></li>
-			  <li><a href="/ui/credentials">My Credentials</a></li>
-              <li><a href="/ui/exports">My Exports</a></li>
-              <li><a href="/admin">Admin</a></li>
-              <li><a href="{% url 'account_logout' %}">{% trans "Logout" %}</a></li>
+          {% if request.user.is_authenticated %} 
+            <ul class="nav navbar-nav">
+	      <li><a href="/ui/collections">Collections</a></li>
```
Comment (justinlittman) [inline sfm/ui/templates/base.html]: These should be reversed urls.

File: sfm/ui/templates/base.html [hunk 2]
```diff
@@ -55,38 +55,37 @@
...
+	      <li><a href="/ui/credentials">Credentials</a></li>
+              <li><a href="/ui/exports">Exports</a></li>
+            </ul>
+            {% endif %}
+         <ul class="nav navbar-nav navbar-right">
+             {% if request.user.is_authenticated %}
+             <li class="dropdown">
+               <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Welcome, {{ user.username }}<span class="caret"></span></a>
+                 <ul class="dropdown-menu">
+                     <!--<li><a href="">Account</a></li> -->
```
Comment (justinlittman) [inline sfm/ui/templates/base.html]: Delete comment.

File: sfm/ui/templates/base.html [hunk 3]
```diff
@@ -55,38 +55,37 @@
...
+                     <li><a href="{% url 'account_logout' %}">{% trans "Log out" %}</a></li>
+                 </ul>
+             </li>
             {% else %}
-              <li><a href="{% url 'account_signup' %}">{% trans "Sign Up" %}</a></li>
-              <li><a href="{% url 'account_login' %}">{% trans "Log In" %}</a></li>
+            <li><a href="{% url 'account_signup' %}">{% trans "Sign up" %}</a></li>
```
Comment (justinlittman) [inline sfm/ui/templates/base.html]: Remove translate.

File: sfm/ui/templates/base.html [hunk 4]
```diff
@@ -95,13 +94,19 @@
...
     </div> <!-- /container -->
 
     {% block modal %}{% endblock modal %}
 
-    <div class="footer" id="footer"></div>
-  </body>
+    <footer>
+      <nav class="navbar navbar-default">
+        <div class="container-fluid">
+            <div class="nav navbar-header"><p>GW Libraries</p></div>
```
Comment (justinlittman) [inline sfm/ui/templates/base.html]: Remove GW Libraries pending support for branding.

**Comment by justinlittman** [comment]
When logging out, get a message that says "You have signed out."

**Pull request merged**